source/common/config/subscription_factory_impl.cc

```diff
@@ -51,15 +51,17 @@ SubscriptionPtr SubscriptionFactoryImpl::subscriptionFromConfigSource(
           local_info_, cm_, api_config_source.cluster_names()[0], dispatcher_, random_,
           Utility::apiConfigSourceRefreshDelay(api_config_source),
           Utility::apiConfigSourceRequestTimeout(api_config_source), restMethod(type_url), type_url,
-          callbacks, stats, Utility::configSourceInitialFetchTimeout(config), validation_visitor_);
+          api_config_source.transport_api_version(), callbacks, stats,
```

Comment:
Are there any tests that use versions other than the AUTO?

Reply:
Sure, test coverage is provided by `api_version_integration_test` already, but I can fill in a few more tests here as well.

Reply:
Oh yes, that test. No need for additional tests, it should be enough.